txnkv/transaction/pessimistic.go

```diff
@@ -96,23 +95,11 @@ func (action actionPessimisticLock) handleSingleBatch(c *twoPhaseCommitter, bo *
 		}
 		mutations[i] = mut
 	}
-	elapsed := uint64(time.Since(c.txn.startTime) / time.Millisecond)
-	ttl := elapsed + atomic.LoadUint64(&ManagedLockTTL)
-	if _, err := util.EvalFailpoint("shortPessimisticLockTTL"); err == nil {
```

Comment:
I think we'd better keep the failpoint and move it into the loop. Some tests may depend on the failpoint.